src/main/java/edu/cornell/kfs/pdp/service/impl/CuPdpEmailServiceImpl.java

```diff
@@ -204,9 +199,9 @@ private MailMessage createAdviceMessageAndPopulateHeader(PaymentGroup paymentGro
             message.setSubject(customer.getAdviceSubjectLine());
         }
         else {
-            message.addToAddress(mailService.getBatchMailingList());
-            message.addCcAddress(mailService.getBatchMailingList());
-            message.addBccAddress(mailService.getBatchMailingList());
+            message.addToAddress(emailService.getDefaultFromAddress());
+            message.addCcAddress(emailService.getDefaultFromAddress());
+            message.addBccAddress(emailService.getDefaultFromAddress());
```

Comment:
Should these three properties be using the default "to" address instead of the default "from" one?